extension-readme.md

```diff
@@ -1,182 +1,242 @@
-1. In the list of AWS credentials profiles that is displayed, choose the AWS credentials profile that you want to use. (For additional information, see [Setting Up Your AWS Credentials](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-credentials.html) in the _AWS Toolkit for Visual Studio Code User Guide_.)
+### **Step 3**: Work with Available AWS Resources
 
-#### Step 3: Work with Available AWS Resources
+For complete information, see [Working with AWS Services](https://docs.aws.amazon.com/console/toolkit-for-vscode/working-with-aws) in the _AWS Toolkit for Visual Studio Code User Guide_ .
 
-1. If the **AWS: Explorer** Side Bar isn't displayed, display it by choosing the **AWS** icon in the Activity Bar. (To hide the **AWS: Explorer**, choose the **AWS** icon again.)
-1. If you have existing resources in your AWS account but they aren't displayed in the **AWS: Explorer**, choose the **Refresh** icon in the **AWS: Explorer** to display them. 
-1. Expand the AWS Region that contains the resource that you want to manage. (To hide or show AWS Region information, choose **View, Command Palette** on the menu bar, and then choose **AWS: Hide region from the Explorer** or **AWS: Show region in the Explorer**.)
-1. Expand the AWS service that contains the resource that you want to manage. (Currently, the Toolkit doesn't support all available AWS services and resource types.)
-1. If applicable, expand the parent resource that contains the child resource that you want to manage.
-1. Right-click the resource to manage, and then choose one of the available actions in the context menu. (Currently, the Toolkit doesn't support all available resource actions.)
+1. If the **AWS Explorer** isn't showing, open it by choosing the **AWS** icon in the Activity Bar:
+   ![AWS Explorer Icon](./resources/marketplace/aws-explorer-icon.png)
+2. If you have existing resources in your AWS account but they aren't being displayed in the **AWS Explorer** (and assuming that your credentials and a connection have been set up properly), choose the **Refresh** icon in the **AWS Explorer** to show them. 
+3. Expand the AWS Region that contains the resource that you want to manage.
 
-For more information, see the following in the _AWS Toolkit for Visual Studio Code User Guide_:
+   (**Note**: To show or hide AWS regions, choose **View, Command Palette** on the menu bar, and then choose **AWS: Show region in the Explorer** or **AWS: Hide region from the Explorer**.)
+4. Expand the supported AWS service that contains the resource that you want to manage.
 
-* [Setting Up Your AWS Credentials](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-credentials.html)
-* [Changing AWS Regions](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-region.html)
-* [Exploring AWS Explorer](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/aws-explorer.html)
+   (**Note**: Not all services are currently supported.)
```

Comment:
Is there a better way to convey this information? (ditto below)

Reply:
I think you mean the **Note**.
I tried to make it more concise so that it wouldn't get in the way of the important stuff, but there might be more we could do.
Maybe add a note at the top (similar to your question about "serverless only at this time) with a general statement (or something like that), then delete these notes?